src/test/article.spec.js

```diff
@@ -1199,3 +1203,92 @@ describe('GET /api/v1/articles/popular', () => {
     findAllStub.restore();
   });
 });
+describe('POST api/v1/:articleId/comments/:commentId/like', () => {
+  beforeEach(async () => {
+    await db.Article.destroy({ cascade: true, truncate: true });
+    await db.User.destroy({ cascade: true, truncate: true });
+    // await db.Commentlike.destroy({ cascade: true, truncate: true });
```

Comment:
You should also delete the `comment` and `commentLike` table to ensure you are running independent tests